Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.11-alpine3.8 as builder
+
+WORKDIR /go/src/github.com/andreas-schroeder/kafka-health-check
+
+ENV CGO_ENABLED 0 
+
+RUN apk update \
+    && apk add make git
+
+RUN go get github.com/kardianos/govendor
+
+COPY . .
+
+RUN make deps && make
+
+FROM scratch
```

Comment:
statically compiling and running from scratch hides any CVE also. 

I think we talked about running amazon base image everywhere. but I guess we can tackle that later

Reply:
Do we want to detect CVEs on runtime? or though buildtime validation (so if we are running version X, we can check the code associated with that build instead of "scanning the running binary")
Also, wasn't the CVE thing the reason we were going away from alpine?

Reply:
I guess we could build our own golang image
However that makes it impossible to contribute back our Dockerfile

Reply:
True.